File: angular-gpp/src/app/models/ItemSolicitacaoCriacaoAstecaModel.ts
export class ItemSolicitacaoCriacaoAstecaModel {
  idItemPedidoSaida: number = 0;
  quantidade: number = 0;
  valor: number = 0;
  peca: string = "";
  separado: number = 0;
  separador: string = "";
  dataSeparacao: Date | undefined;
  motivoTrocaPeca: string = "";
  pendenciaItem: boolean = false;
  endereco: string = "";
  saldo_disponivel: number = 0;
  pecaEstoque: string = "";

  

}

File: gpp-api/src/main/java/com/develop/gpp/domain/entity/ItemSolicitacaoCriacaoAstecaModel.java
package com.develop.gpp.domain.entity;

import java.util.Date;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
public class ItemSolicitacaoCriacaoAstecaModel {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long idItemPedidoSaida;
    
    private int quantidade;
    
    private double valor;
    
    private String peca;
    
    private int separado;
    
    private String separador;
    
    private Date dataSeparacao;
    
    private String motivoTrocaPeca;
    
    private boolean pendenciaItem;
    
    private String endereco;
    
    private double saldo_disponivel;
    
    private String pecaEstoque;
    
    // Add getters and setters
}
